src/palindrom-dom.js

```diff
@@ -236,24 +211,9 @@ const PalindromDOM = (() => {
     }
   }
 
-  PalindromDOM.prototype = Object.create(Palindrom.prototype);
-
-  /* backward compatibility, not sure if this is good practice */
-  if (typeof global === 'undefined') {
-    if (typeof window !== 'undefined') {
-      /* incase neither window nor global existed, e.g React Native */
-      var global = window;
-    } else {
-      var global = {};
-    }
-  }
-  global.PuppetDOM = PalindromDOM;
-
   /* Since we have Palindrom bundled,
   let's expose it in case anyone needs it */
-  global.Puppet = Palindrom;
-  global.Palindrom = Palindrom;
-
+  PalindromDOM.Palindrom = Palindrom;
```

Comment:
What's the practical use of this?

Reply:
Well, when you have PalindromDOM loaded, it means you paid for Palindrom + PalindromDOM. So I thought it would be nice to give access to Palindrom. It's harmless and is potentially useful. I even think exposing jsonpatch's `applyPatch` would be very useful and free.

Reply:
But you have access to Palindrom since PalindromDOM extends Palindrom.

Reply:
How? `Palindorm` is undefined when you import `PalindromDOM`.

Reply:
`PalindromDOM` extends `Palindrom` so all the public methods of `Palindrom` are accessible on `PalindromDOM`.

Reply:
Well true but if you construct `PalindromDOM` you execute this code:

```js
      this.listening = true;
      this.element.addEventListener('click', this.clickHandler);
      window.addEventListener('popstate', this.historyHandler); //better here than in constructor, because Chrome triggers popstate on page load

      this.element.addEventListener(
        'palindrom-morph-url',
        this.morphUrlEventHandler
      );

      this.element.addEventListener(
        'palindrom-redirect-pushstate',
        this.historyHandler
      );
```

Reply:
If someone wants to use Palindrom without PalindromDOM, they should import Palindrom. I don't really see who could use `PalindromDOM.Palindrom` for a reason other than a mistake. Can you give some use cases?

Reply:
I couldn't come up with any. I'll remove it.